### PR TITLE
DOC-3714, DOC-3682 "Home" page -> "Course" page

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3682](https://openedx.atlassian.net/browse/DOC-3682)

Replace references to "Home" page with "Course" page in all guides, correcting/rewriting content and replacing screenshots where necessary.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Dev FYI: @robrap 
- [x] Doc team review: @edx/doc
- [ ] Product review: @marcotuts 

FYI: @dhixonedx

### Testing
- [x] Ran ./run_tests.sh without warnings or errors - one unrelated error to be fixed by @srpearce in a separate PR

### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/course_assets/handouts_updates.rst [hunk 1]
```diff
@@ -45,7 +51,7 @@ To add a course update, follow these steps.
      .. note::
         To make sure that your announcements are accessible, if you use
         headings inside an announcement, make sure that you use only level 6
-        headings (``<h6>``). The structure of the course **Home** page uses
+        headings (``<h6>``). The structure of the course **Course** page uses
```
Comment (catong) [inline en_us/shared/course_assets/handouts_updates.rst]: @robrap or @dkh is this statement also correct for the **Course** page?

File: en_us/shared/course_assets/handouts_updates.rst [hunk 2]
```diff
@@ -22,18 +25,21 @@ course updates and handouts on the **Home** page in your course.
 Add a Course Update
 **********************
 
-You add updates to notify learners of upcoming exams or deadlines, changes in
-the course schedule, or to make other announcements. Updates are visible to
-enrolled learners as soon as you post them.
+You add updates to welcome learners as they start the course, notify learners
```
Comment (catong) [inline en_us/shared/course_assets/handouts_updates.rst]: @marcotuts Can you please review this topic to verify that behavior (e.g. around visibility) is still as described? Thank you!

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 3]
```diff
@@ -417,9 +417,9 @@ interpret this message, note that:
   recipients you selected who were enrolled in the course (and had activated
   their user accounts) when you sent the email message.
 
-  Note that the **Total number of enrollees** shown on the **Home** page
-  includes all currently enrolled learners, regardless of account activation
-  status, and as a result can be different than this number.
+  Note that the **Total number of enrollees** shown in enrollment data in
```
Comment (catong) [inline en_us/shared/manage_live_course/bulk_email.rst:420]: @dsjen Would you mind reviewing this paragraph? I believe that the original text referred to the Insights Home page - does it make sense as I've rewritten it?

Comment (dsjen) [reply en_us/shared/manage_live_course/bulk_email.rst:420]: Thanks for the update! This makes sense.

Comment (catong) [reply en_us/shared/manage_live_course/bulk_email.rst:420]: Thank you!

File: en_us/shared/course_assets/pages.rst [hunk 4]
```diff
@@ -30,28 +29,32 @@ In the LMS, learners see these options in the navigation bar at the top of
...
+**Instructor** option in the navigation bar. Learners do not see the
+**Instructor** option.
+
+To customize your course, you can make the following changes to the pages that
+the system adds by default.
 
 * You can :ref:`change the order<Reorder Pages>` in which the pages are
-  presented. The exceptions are the **Home** and **Course** pages, which are
-  always presented in the first and second positions.
+  presented, with the exception that the **Course*pages is always presented in
```
Comment (grantgoodmanedX) [inline en_us/shared/course_assets/pages.rst]: **Course** page

File: en_us/shared/glossary/glossary.rst [hunk 5]
```diff
@@ -248,11 +248,18 @@ C
...
-  the **Course** page. This page was formerly called the **Courseware** page.
+  The page that opens first when learners access your course. On the **Course**
+  page, learners can view the course outline and directly access the course,
+  either by clicking a specific section or subsection on the outline, or by
+  clicking the **Start Course** button (**Resume Course** if the learner has
+  previously accessed the course).
+
+  The latest course update, such as a course welcome message, appears above the
+  course outline. Links to various **Course Tools** including **Bookmarks**,
+  **Reviews** and **Updates** appears at the side of this page. This page is a
```
Comment (grantgoodmanedX) [inline en_us/shared/glossary/glossary.rst]: appear, not appears

**Review by grantgoodmanedX — APPROVED** [review]
Two minor edits. It otherwise looks good, and covers all the relevant search hits for Home that I found.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1553

**Pull request merged**